pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -414,11 +398,21 @@ func (alloc *Action) allocateResourcesForTasks(tasks *util.PriorityQueue, job *a
 			}
 		}
 
+		task.JobAllocatedHyperNode = jobAllocatedNewHyperNode
```

Comment:
Does the modification in `allocateResourcesForTasks` func just want to get the hyperNode name of current task, if that's true, I think we should implement this in somewhere else because there is already the same logic in `allocateResourceForTasksWithTopology`, for instance we can abstract this in `allocateResources`.